tools/bridge/tests/conftest.py

```diff
@@ -106,6 +111,32 @@ def proxy_validator_keys(proxy_validator_accounts_and_keys):
     return keys
 
 
+@pytest.fixture
+def validator_account_and_key(proxy_validator_accounts_and_keys):
+    """Address and private key of the validator running the confirmation sender."""
+    accounts, keys = proxy_validator_accounts_and_keys
+    return accounts[0], keys[0]
+
+
+@pytest.fixture
+def validator_address(validator_account_and_key):
+    """Address of the validator running the confirmation sender."""
+    account, _ = validator_account_and_key
+    return account
+
+
+@pytest.fixture(scope="session")
+def non_validator_address(accounts, number_of_validators):
```

Comment:
maybe use the same pattern as in `validator_account_and_key` to ensure that `validator_key` and `validator_address` always belong to the same account.

Reply:
:heavy_check_mark: